flatdata-go/flatdata/Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+
+# Gopkg.toml example
```

Comment:
Here, the same. Remove the example lines?

Reply:
Same as previous